tests/testthat/test-scoreSingleSamples.R

```diff
@@ -12,10 +12,9 @@ test_that("scoreSingleSamples can use genesets of size n = 1 and value is gene",
 test_that("do.scoreSingleSamples.gsva produces correct gsva,plage,ssGSEA scores", {
   vm <- exampleExpressionSet()
   gdb <- conform(exampleGeneSetDb(), vm)
-  lol <- as.list(gdb)
+  gs.idxs <- as.list(gdb, active.only = TRUE, value = "x.idx")
+  lol <- lapply(gs.idxs, function(i) rownames(vm$E)[i])
```

Comment:
Thank you for the suggestion and bringing the potential fix for bioc to my attention, @gladkia !

Sorry, I don't have a bioc-devel environment setup just yet, but if your proposal fixes the error, we should be able to replace this:

```
lol <- as.list(gdb)
```

with this expanded one liner

```
lol <- as.list(gdb, active.only = TRUE, value = "feature_id")
```

I will check and see ...

Reply:
@lianos that's great. 
The code was based on this [chunk](https://github.com/lianos/sparrow/blob/devel/R/scoreSingleSamples.R#L280-L285).